nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -6,17 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use the Node Maintenance Operator to place nodes in maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of {VirtProductName} installation.
+You can use the Node Maintenance Operator to place nodes in maintenance mode.
```

Comment:
> > I would like to delete the second sentence in the beginning of the page - http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html `This is a standalone version of the Node Maintenance Operator that is independent of OpenShift Virtualization installation.`
> 
> @razo7 This is now updated, see http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html

Now the `You can use the Node Maintenance Operator to place nodes in maintenance mode.` sentence doesn't add anything, that we don't say under `About the Node Maintenance Operator`.
I think it would be better to delete the whole sentence and the about section, `About the Node Maintenance Operator`,  and let the page starts with `You can place nodes into maintenance mode using the oc adm utility, or using NodeMaintenance custom resources (CRs).`

Reply:
Updated as requested http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html